docs/node-favorite.directive.md

```diff
@@ -0,0 +1,83 @@
+# Node Favorite directive
+
+Selectively toggle nodes as favorite
+
+<!-- markdown-toc start - Don't edit this section.  npm run toc to generate it-->
+
+<!-- toc -->
+
+- [Basic Usage](#basic-usage)
+  * [Properties](#properties)
+  * [Events](#events)
+- [Details](#details)
+
+<!-- tocstop -->
+
+<!-- markdown-toc end -->
+
+## Basic Usage
+
+```html
+<adf-toolbar>
+    <button md-icon-button
+            (adf-toggle-favorite)="done()"
```

Comment:
in the doc the event is selection-toggle-favorite but I think we can change it across all the implementation and use just (change or toggle)  we should try to favor simple event name and not use the kebabcase. 

https://github.com/Alfresco/alfresco-ng2-components/wiki/Code-Style